### ISSUE TITLE
 DBMS_OUTPUT.PUT_LINE and PRINT

### ISSUE THREAD

**Issue by jyf1997** [dbeaver/dbeaver#22510]
I added the DM database (a development database based on Oracle) in the driver manager. However, when executing DBMS_OUTPUT.PUT_LINE and PRINT statements, the output results are not displayed on the output console. How can I adjust this?

**Comment by LonwoLonwo** [comment]
Hello @jyf1997 

How exactly did you add your driver via driver manager? Did you add it with a generic type or as a copy of the Oracle driver?
Could you please provide a screen of your driver settings?
And a link on your database.

**Comment by jyf1997** [comment]
> 你好
> 
> 您究竟是如何通过驱动程序管理器添加驱动程序的？是将其添加到泛型类型还是作为 Oracle 驱动程序的副本？您能否提供驱动程序设置的屏幕？以及数据库上的链接。

![Uploading N%@$_4715GH5~{KNRLT34@P.png…]()

**Comment by jyf1997** [comment]
> Hello @jyf1997
> 
> How exactly did you add your driver via driver manager? Did you add it with a generic type or as a copy of the Oracle driver? Could you please provide a screen of your driver settings? And a link on your database.
![N%@$_4715GH5~{KNRLT34@P](https://github.com/dbeaver/dbeaver/assets/24623258/3474be6e-b695-4059-9b4b-c6f771bcf5df)

**Comment by LonwoLonwo** [comment]
Then, you could try to add it as an Oracle copy driver. 

Or check this setting status:

![2024-01-23 17_19_35-Preferences](https://github.com/dbeaver/dbeaver/assets/45152336/f1d3035d-123c-421c-b0fe-fd5d24e1ca5a)

And one more thing - did you try to use the Dameng connector?

**Comment by jyf1997** [comment]
> 然后，您可以尝试将其添加为 Oracle 复制驱动程序。
> 
> 或检查此设置状态：
> 
> ![2024-01-23 17_19_35-首选项](https://private-user-images.githubusercontent.com/45152336/298997995-f1d3035d-123c-421c-b0fe-fd5d24e1ca5a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYwNTg2OTIsIm5iZiI6MTcwNjA1ODM5MiwicGF0aCI6Ii80NTE1MjMzNi8yOTg5OTc5OTUtZjFkMzAzNWQtMTIzYy00MjFjLWIwZmUtZmQ1ZDI0ZTFjYTVhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTI0VDAxMDYzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEzNjQyMTEyYjk2MWY0YTFkOGE1OTNkYjQxYTlhOTljN2M4NjViZTUxODRkZGVhZTE2ZGUxNTA4MjI2NGE0ZmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ZK0rwiWzp3ebeOasSIAmzqyG5-3Rf0RSRboefwHiI1Y)
> 
> 还有一件事 - 您是否尝试使用Dameng连接器？

Dameng's DBA tool can be used to output normally, but I want to use dbeaver to complete subsequent DBA operation and maintenance work. The switch you mentioned is on. There is also a copy of the oracle driver, and the url template cannot be modified.

**Comment by LonwoLonwo** [comment]
Ok, I need to be more specific:
did you try to connect via the Dameng connector in DBeaver?
If it is working fine for you and one thing is missed only - is output - then I'll classify this ticket as a feature request.

![2024-01-24 10_58_27-Connect to a database](https://github.com/dbeaver/dbeaver/assets/45152336/1862bbd3-aed2-4b0c-b84e-12bc9e4871c1)

**Comment by jyf1997** [comment]
> Ok, I need to be more specific: did you try to connect via the Dameng connector in DBeaver? If it is working fine for you and one thing is missed only - is output - then I'll classify this ticket as a feature request.
> 
> ![2024-01-24 10_58_27-Connect to a database](https://private-user-images.githubusercontent.com/45152336/299253361-1862bbd3-aed2-4b0c-b84e-12bc9e4871c1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYwODc1MDUsIm5iZiI6MTcwNjA4NzIwNSwicGF0aCI6Ii80NTE1MjMzNi8yOTkyNTMzNjEtMTg2MmJiZDMtYWVkMi00YjBjLWI4NGUtMTJiYzllNDg3MWMxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTI0VDA5MDY0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMxNWJkZWNhYTQwZGVkMzJkZTJmZmQ4MmYwMjJmMGVmYWY5MDNhMWUyOGRlNGE4YzNmYTVlYWU5MTNhYTVjMWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.rAhDeURj9R6jNITz1GmouOF2fjP1NAeCmFek0eQVlbs)

Yes, after I created a new dameng driver in the driver manager, the new connection execution output statement was not displayed in the output console. The same statement is displayed in the Dameng DBA management tool. Executing the same output statement with the Oracle library in dbeaver can also be displayed normally. These are some of the attempts I've made.

**Comment by LonwoLonwo** [comment]
Ok. Thanks for the request, then.
